tests/conftest.py

```diff
@@ -20,36 +20,39 @@ def capture_logs(caplog):
 
 
 @pytest.fixture
-def example_model(tmp_path):
+def tmp_model(tmp_path):
+   """Example resqpy model in a temporary directory unique to each test"""
+
+   model = Model(create_basics=True)
+   model_path = str(tmp_path / 'tmp_model.epc')
+   model.set_epc_file_and_directory(model_path)
+   model.create_hdf5_ext(add_as_part=True, file_name=model_path[:-4] + '.h5')
+   return model
+
+
+@pytest.fixture
+def example_model_and_crs(tmp_model):
    """ Returns a fresh RESQML Model and Crs, in a temporary directory """
```

Comment:
Not sure this docstring is entirely accurate now that the tmp_model is being passed in